File: public/examples/integrations/toolkits/stripe/create_invoice_item_example_call_tool.js
import { Arcade } from "@arcadeai/arcadejs";

const client = new Arcade(); // Automatically finds the `ARCADE_API_KEY` env variable

const USER_ID = "{arcade_user_id}";
const TOOL_NAME = "Stripe.CreateInvoiceItem";

const toolInput = {
  customer: "cus_123456789",
  price: "price_123456789",
  invoice: "in_123456789"
};

const response = await client.tools.execute({
  tool_name: TOOL_NAME,
  input: toolInput,
  user_id: USER_ID,
});

console.log(response);

File: public/examples/integrations/mcp-servers/stripe/create_invoice_item_example_call_tool.py
from arcadepy import Arcade

client = Arcade()  # Automatically finds the `ARCADE_API_KEY` env variable

USER_ID = "{arcade_user_id}"
TOOL_NAME = "Stripe.CreateInvoiceItem"

tool_input = {
    "customer": "cus_123456789",
    "price": "price_123456789",
    "invoice": "in_123456789"
}

response = client.tools.execute(
    tool_name=TOOL_NAME,
    input=tool_input,
    user_id=USER_ID,
)
print(response)
